src/nylas-connection.ts

```diff
@@ -309,7 +309,14 @@ export default class NylasConnection {
             ) {
               return resolve(response.text());
             } else {
-              return resolve(response.json());
+              return response
+                .json()
+                .then(json => {
+                  return resolve(json);
+                })
+                .catch(() => {
+                  return resolve(undefined);
+                });
```

Comment:
https://stevenklambert.com/writing/fetch-json-text-fallback/
This might not work!

Reply:
Great find @AaronDDM, thank you! Fetch has been very frustrating to deal with regarding this.

Reply:
@AaronDDM after doing some research looks like the article's right. just added in the new clone method. Thanks for spotting this!